src/Compilers/VisualBasic/Portable/Symbols/Attributes/AttributeData.vb

```diff
@@ -376,7 +383,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
 
                     If resolvedFilePath Is Nothing Then
 
-                        ' BC31210: Unable to resolve file path '{0}' specified for the named argument '{1}' for PermissionSet attribute
+                        ' BC31216: Unable to resolve file path '{0}' specified for the named argument '{1}' for PermissionSet attribute.
                         Dim argSyntaxLocation As Location = If(arguments.AttributeSyntaxOpt IsNot Nothing,
                                                                arguments.AttributeSyntaxOpt.ArgumentList.Arguments(1).GetLocation(),
                                                                NoLocation.Singleton)
```

Comment:
I wasn't able to write a test that makes a crash here, so I didn't make a change.
In case someone was able to, let me know.